main_routers/characters_router.py

```
@@ -93,6 +93,8 @@ def _validate_profile_name(name: str) -> str | None:
         return '档案名不能包含点号(.)'
     if result.code == 'reserved_device_name':
         return '档案名不能使用 Windows 保留设备名'
+    if result.code == 'reserved_route_name':
+        return '此名称是系统保留的路由名称，不能用作档案名'
```

Ruff (0.15.9)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)